staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -200,7 +208,13 @@ func (d *DiscoveryClient) ServerResourcesForGroupVersion(groupVersion string) (r
 	resources = &metav1.APIResourceList{
 		GroupVersion: groupVersion,
 	}
-	err = d.RESTClient().Get().AbsPath(url.String()).Do().Into(resources)
+
+	if !tenantedFormat {
+		err = d.RESTClient().Get().AbsPath(url.String()).Do().Into(resources)
+	} else {
+		err = d.RESTClient().Get().AbsPath(url.String()).Param("tenant", parts[0]).Do().Into(resources)
```

Comment:
string constant "tenant" is also used at another place. Is it possible to put this into a shareable constant?

Reply:
Fixed.